### PR TITLE
Added "universal-apple-macos11-64" descriptions opsys.ttl file ...

### PR DESCRIPTION
**Pull request by HughWilliams**

### CODE REVIEW

File: opsys.ttl [hunk 1]
```diff
@@ -11078,4 +11078,29 @@ source:
...
+		<http://data.openlinksw.com/oplweb/processor/x86_64/2#this> ,
+		<http://data.openlinksw.com/oplweb/processor/M1/2#this> ;
+    oplsof:OperatingSystemSupported "1"^^xsd:boolean ;
+    oplsof:OperatingSystemUpwardCompatible "1"^^xsd:boolean ;
+    oplsof:OperatingSystemVersion "11.x" ;
+    oplcomp:hasComponent
+        <http://data.openlinksw.com/oplweb/component/universal-apple-macos11-64-presto-odbcexp-clnt-only_mv#this>,
+        <http://data.openlinksw.com/oplweb/component/universal-apple-macos11-64-presto-odbclt-clnt-only_mv#this>,
+        <http://data.openlinksw.com/oplweb/component/universal-apple-macos11-64-presto-odbclt-clnt-only_sv#this> ;
+    oplpro:hasProduct <http://data.openlinksw.com/oplweb/product/odbc-presto-ee#this> ;
```
Comment (TallTed) [inline opsys.ttl:11103]: I think these Presto drivers, Lite and Express, are no longer being produced.

File: opsys.ttl [hunk 2]
```diff
@@ -182,8 +182,8 @@ source:
               <http://data.openlinksw.com/oplweb/opsys/universal-apple-macosx10.12-32#this> ,
               <http://data.openlinksw.com/oplweb/opsys/i686-apple-macosx10.13-32#this> ,
               <http://data.openlinksw.com/oplweb/opsys/x86_64-apple-macosx10.13-64#this> ,
-              <http://data.openlinksw.com/oplweb/opsys/universal-apple-macosx10.13-32#this> .
-
+              <http://data.openlinksw.com/oplweb/opsys/universal-apple-macosx10.13-32#this> ,
+	      <http://data.openlinksw.com/oplweb/opsys/universal-apple-macos11-64#this> .
```
Comment (TallTed) [inline opsys.ttl:186]: Given, and based on, existing listings, I think this list (and descriptions below) should also include `i686-apple-macosx10.14-32`, `x86_64-apple-macosx10.14-64`, and `x86_64-apple-macosx10.15-64`.

Possibly also `universal-apple-macosx10.14-32`, `x86_64-apple-macos11-64`, and `arm64-apple-macos11-64`.

File: opsys.ttl [hunk 3]
```diff
@@ -33,7 +33,7 @@ source:
    schema:name "OpenLink Operating Systems"^^xsd:string ;
    schema:comment """This is a turtle document that uses Linked Data oriented content to describe OpenLink Operating Systems"""@en ;
    schema:dateCreated "2013-01-01T13:00:00-05:00"^^xsd:dateTime ;
-   schema:dateModified "2017-12-18T17:00:00-05:00"^^xsd:dateTime ;
+   schema:dateModified "2021f-06-02T17:00:00-05:00"^^xsd:dateTime ;
```
Comment (TallTed) [inline opsys.ttl:36]:
```suggestion
   schema:dateModified "2021-06-02T17:00:00-05:00"^^xsd:dateTime ;
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Comment by openlink** [comment]
Just withdraw this PR since it has been superseded.

**Pull request closed without merging**